src/main/java/org/ow2/proactive/connector/iaas/cloud/provider/azure/AzureProvider.java

```diff
@@ -240,23 +261,27 @@
                                                                             })
                                                                             .collect(Collectors.toList());
 
-        // Create all VMs in parallel and collect IDs
+        // Execute init scripts asynchronously on VMs
+        //vms.forEach(vm -> executeScriptOnVM(vm, instance.getInitScript()));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)